src/views/Consultation.vue

```diff
@@ -3,21 +3,33 @@
     <p>STEP3</p>
     <h3>ご相談内容をご記入ください</h3>
     <p>ご相談内容</p>
-    <textarea rows="8" cols="80"></textarea>
+    <textarea rows="8" cols="80" v-model="textContent" @input="checkText"></textarea>
     <nav>
       <router-link to="/Questionnaire"><button>戻る</button></router-link>
-      <button>進む</button>
+      <router-link to="/Check"><button>進む</button></router-link>
     </nav>
   </div>
 </template>
 
 <script>
+  import { mapGetters } from 'vuex'
+
   export default {
     name: 'consultations',
     data: () => {
       return{
-
+        textContent: ''
+      }
+    },
+    methods: {
+      checkText: function(){
```

Comment:
functionを省略する書き方もあるので覚えておきましょう。
```suggestion
      checkText() {
```